src/models/dft.js

```diff
@@ -14,94 +11,31 @@ export class DFTModel extends Model {
      * @summary Build slug string based on model information
      * @returns {string}
      */
-    get groupSlug() {
-        return [
-            this._application.shortName,
-            this.type,
-            this.subtype,
-            this.functional.slug,
-            this.refiners.map(o => o.slug).join("+"),
-            this.modifiers.map(o => o.slug).join("+"),
-        ].join(":").replace("::", ":").replace(/:$/, "");
-    }
-
-    get defaultFunctional() {
-        return treeSlugToNamedObject(this.treeBranchForSubType.functionals[0])
+    buildGroupSlug(application) {
```

Comment:
We should consider adjusting where `groupSlug` is used

Reply:
I added a todo with references to the usage of `groupSlug`.

Reply:
I don't understand - where exactly are we planning to introduce that change? Seems like a straightforward change to add to webapp as part of this task, or not?

Reply:
My bad, I created a web-app branch now.

By the way, since the `Model` will be a compound model, there are potentially several group slugs stemming from the unit models. I added a function to the compound model to concatenate all unit model group slugs.

Reply:
Good thought. We might need to figure out some way to assert that the order of units is properly accounted for in the future also.